### PR TITLE
Getting Started Clarifications and  Example Code File

### PR DESCRIPTION
**Pull request by masstep**
As part of a computer science course, I recently followed the spotipy getting started and made notes of any issues I ran into. I have now added comments in areas that caused me trouble while trying to run some of the features. 

initially I had issues with installing and accessing the spotipy library with the listed installation instruction. 

I also noticed not all of the same information is provided in the github repo as on the website under getting started. I understand the the repo points to the website for extensive information, however the README lacks any mention of how to get the URI redirect for Authorization Code Flow. 

I had issues with the add_track_to_playlist function, but came across the user_playlist_add_tracks function on the website. When I tried to run this function I was informed in pyCharm to use playlist_add_items instead. Neither of the latter 2 functions are in the examples folder, so I thought it would be good to have an alternate example for adding tracks to playlists.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,12 +14,20 @@ Spotipy's full documentation is online at [Spotipy Documentation](http://spotipy
 pip install spotipy
 ```
 
+alternatively 
+
+```bash
+py -m pip install spotipy
```
Comment (stephanebruckert) [inline README.md:20]: what is `py`?

File: README.md [hunk 2]
```diff
@@ -14,12 +14,20 @@ Spotipy's full documentation is online at [Spotipy Documentation](http://spotipy
...
+```
+
 or upgrade
 
 ```bash
 pip install spotipy --upgrade
 ```
 
+After installation if your IDE can not find the libary, add an intepretor path to the library in your IDE.
```
Comment (stephanebruckert) [inline README.md:29]: this is not spotipy related. One is expected to be able to set up python libraries before reading this readme

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+-   Added alternative module installation instruction to README
+-   Added Comment to README - Getting Started for user to add URI to app in Spotify Developer Dashboard. 
+-   Added playlist_add_tracks.py to example folder
```
Comment (stephanebruckert) [inline CHANGELOG.md:12]: please update and move these to the `added` category

File: README.md (hunk 1 above)
Comment (masstep) [reply README.md:20]: Py is the python launcher for windows, where python is the python executable. I could not install Spotipy using python.

File: README.md (hunk 2 above)
Comment (masstep) [reply README.md:29]: That makes sense. I  ran into problems when I successfully installed the library but ran into ModuleNotFoundError when trying to use it which took a lot of internet searching to resolve. I don't have a lot of experience installing libraries.

**Review by stephanebruckert — APPROVED** [review]

**Pull request merged**